components/script/dom/blob.rs

```diff
@@ -166,6 +166,11 @@ impl Blob {
         }
     }
 
+    /// Get a copy of the type_string
+    pub fn get_type_string(&self) -> Vec<u8> {
```

Comment:
Let's make this return the string value, and have the caller convert it into a `Vec<u8>` instead.